Solutions/Lookout/Data Connectors/Lookout_API_FunctionApp.json

```diff
@@ -3,7 +3,7 @@
 	"title": "Lookout",
 	"publisher": "Lookout",
 	"descriptionMarkdown": "The [Lookout](https://lookout.com) data connector provides the capability to ingest [Lookout](https://mtp.lookout.com/) events into Azure Sentinel through the REST API. Refer to [API documentation](https://mtp.lookout.com) for more information. The connector provides ability to get events which helps to examine potential security risks and more.",
-	"additionalRequirementBanner": "These queries and workbooks are dependent on a parser based on Kusto to work as expected. ​Follow the steps to use this Kusto functions alias **LookoutEvents** in queries and workbooks [Follow steps to get this Kusto functions>](https://aka.ms/sentinel-lookoutapi-parser).",
+	"additionalRequirementBanner": "These queries and workbooks are dependent on a parser based on Kusto to work as expected. ​Follow the steps to use this Kusto functions alias **LookoutEvents** in queries and workbooks [Follow steps to get this Kusto functions>](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Lookout/Parsers/LookoutEvents.txt).",
```

Comment:
@rajendrakhabiyagate6  I have created short link for parser. https://aka.ms/sentinel-lookoutapi-parser
Please update additionalRequirementBanner filed value as follows.
` This data connector depends on a parser based on a Kusto Function to work as expected [**LookoutEvents**](https://aka.ms/sentinel-lookoutapi-parser) which is deployed with the Azure Sentinel Solution.`